base/generator.jl

```diff
@@ -73,17 +73,19 @@ Given the type of an iterator, return one of the following values:
    for the iterator.
 * `IsInfinite()` if the iterator yields values forever.
 
-The default value (for iterators that do not define this function) is `HasLength()`.
+The default value (for iterator types that do not define this function) is `HasLength()`.
 This means that most iterators are assumed to implement [`length`](@ref).
+An example of changing this default:
+```Base.IteratorSize(::Type{MyIteratorType}) = SizeUnknown()```
```

Comment:
Might want to put Type{<:MyIteratorType} in case MyIteratorType has type parameters. That's also a common mistake.

Reply:
Good point! Done.